main.go

```diff
@@ -11,30 +14,101 @@ const (
 
 // generateRandomElements generates random elements.
 func generateRandomElements(size int) []int {
-	// ваш код здесь
+	if size <= 0 {
+		return []int{}
+	}
+
+	data := make([]int, size)
+
+	for i := range data {
+		data[i] = rand.Intn(1_000_000_000_000)
+	}
+
+	return data
 }
 
 // maximum returns the maximum number of elements.
 func maximum(data []int) int {
-	// ваш код здесь
+	if len(data) == 0 {
+		panic("Empty massive")
```

Comment:
Паника используется в исключительных случаях, а тут просто нет данных, значит 0
```suggestion
		return 0
```